mlflow/models/docker_utils.py

```diff
@@ -44,41 +43,49 @@
 """
 
 
-def _get_mlflow_install_step(dockerfile_context_dir, mlflow_home):
+def _get_mlflow_install_step(dockerfile_context_dir, mlflow_home, no_java=False):
     """
     Get docker build commands for installing MLflow given a Docker context dir and optional source
     directory
     """
     if mlflow_home:
         mlflow_dir = _copy_project(
             src_path=mlflow_home, dst_path=dockerfile_context_dir)
+        java_commands = ("RUN cd /opt/mlflow/mlflow/java/scoring && "
```

Comment:
Rather than overwriting can this be formatted as 

```
java_commands = "" if no_java else "...."
```

or a simple `if` ... `else`, construct. Same comment for `java_commands` usage below.